grails-app/conf/application.yml

```diff
@@ -8,7 +8,7 @@
 
 ---
 grails:
```

Comment:
Need to create a new ticket to make sure we've migrated everything from Config.groovy over to application.yml.